Cannot run blog demo on Windows

# Bug report

## Describe the bug

When running `yarn demo blog dev`, this happens:

```sh
C:\Users\exodi\Documents\keystone-5>yarn demo blog dev
yarn run v1.19.1
$ yarn --cwd ./demo-projects blog dev
$ yarn --cwd blog dev
$ cross-env NODE_ENV=development DISABLE_LOGGING=true node -r dotenv-safe/config `yarn bin`/keystone dev
internal/modules/cjs/loader.js:797
    throw err;
    ^

Error: Cannot find module 'C:\Users\exodi\Documents\keystone-5\demo-projects\blog\`yarn'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:794:15)
    at Function.Module._load (internal/modules/cjs/loader.js:687:27)
    at Function.Module.runMain (internal/modules/cjs/loader.js:1025:10)
    at internal/main/run_main_module.js:17:11 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Presumably this is because the backtick command insertion ( ``` `yarn bin`/keystone dev```) doesn't work in cmd.exe (which I'm using). ☹️ 

Even if I replace it with the proper path manually, this still happens:
```sh
C:\Users\exodi\Documents\keystone-5>yarn demo blog dev
yarn run v1.19.1
$ yarn --cwd ./demo-projects blog dev
$ yarn --cwd blog dev
$ cross-env NODE_ENV=development DISABLE_LOGGING=true node -r dotenv-safe/config ../../node_modules/.bin/keystone dev
C:\Users\exodi\Documents\keystone-5\node_modules\.bin\keystone:2
basedir=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")
          ^^^^^^^

SyntaxError: missing ) after argument list
```

## System information

- OS: Windows 10 1903

--- a/.changeset/loud-fans-vanish.md
+++ b/.changeset/loud-fans-vanish.md
@@ -1,0 +1,5 @@
+---
+'@keystonejs/demo-project-blog': patch
+---
+
+fix startup issue on Windows

--- a/demo-projects/blog/package.json
+++ b/demo-projects/blog/package.json
@@ -9,9 +9,9 @@
     "node": ">=8.4.0"
   },
   "scripts": {
-    "dev": "cross-env NODE_ENV=development DISABLE_LOGGING=true node -r dotenv-safe/config `yarn bin`/keystone dev",
-    "build": "cross-env NODE_ENV=production node -r dotenv-safe/config `yarn bin`/keystone build",
-    "start": "cross-env NODE_ENV=production node -r dotenv-safe/config `yarn bin`/keystone start"
+    "dev": "cross-env NODE_ENV=development DISABLE_LOGGING=true keystone dev",
+    "build": "cross-env NODE_ENV=production keystone build",
+    "start": "cross-env NODE_ENV=production keystone start"
   },
   "dependencies": {
     "@arch-ui/layout": "^0.2.4",
@@ -35,6 +35,7 @@
     "apollo-upload-client": "^10.0.0",
     "cross-env": "^5.2.0",
     "date-fns": "^1.30.1",
+    "dotenv": "^8.0.0",
     "express": "^4.17.1",
     "graphql-tag": "^2.10.1",
     "isomorphic-unfetch": "^3.0.0",

--- a/demo-projects/blog/schema.js
+++ b/demo-projects/blog/schema.js
@@ -1,3 +1,4 @@
+require('dotenv').config();
 const {
   File,
   Text,

